Vladyslav_Gurevych/HW8/MINorMAX.py

```diff
@@ -0,0 +1,44 @@
+def own_max(data: list, amount_of_result=1):
+    """
+       This function takes arguments(list and amount_of_result) and returns a list with a max value.
+       The number of output values can be changed by entering amount_of_result.
+    """
+    result = []
+    temp = data[:]
+    for count in range(amount_of_result):
```

Comment:
count never used. When we have such variable in loop , we should use _ to show this

for _ in range(amount_of_result):

Reply:
fixed